admin/mirage/serializers/campaign-participation.js

```diff
@@ -0,0 +1,7 @@
+import ApplicationSerializer from './application';
+
+const _includes = [];
+
+export default ApplicationSerializer.extend({
+  include: _includes,
```

Comment:
nitpick: pour être homogene avec les autre fichiers
```suggestion
const include = [];

export default ApplicationSerializer.extend({
  include,
```